### PR TITLE
Update ANSSI profile descriptions

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update title of RHEL7 ANSSI profiles to `BP-028`
  - For compatibility reasons their IDs will remain `nt28`.
- Update descriptions of RHEL ANSSI profiles.
- Update title of `controls/anssi.yml` file.
- The High Hardening levels are still in DRAFT state.

#### Rationale:

- The RHEL profiles are moving to alignment with ANSSI BP-028

### CODE REVIEW

**Comment by yuumasato** [comment]
@fduthilleul Hi, pinging you for your awareness and kind request for review.

File: rhel7/profiles/anssi_nt28_enhanced.profile [hunk 1]
```diff
@@ -1,9 +1,15 @@
 documentation_complete: true
 
-title: 'DRAFT - ANSSI DAT-BP28 (enhanced)'
+title: 'ANSSI BP28 (enhanced)'
```
Comment (mildas) [inline rhel7/profiles/anssi_nt28_enhanced.profile:3]: `title: 'ANSSI BP-028 (enhanced)'`

File: rhel7/profiles/anssi_nt28_high.profile [hunk 2]
```diff
@@ -1,9 +1,15 @@
 documentation_complete: true
 
-title: 'DRAFT - ANSSI DAT-BP28 (high)'
+title: 'DRAFT - ANSSI BP28 (high)'
```
Comment (mildas) [inline rhel7/profiles/anssi_nt28_high.profile:3]: `title: 'DRAFT - ANSSI BP-028 (high)'`

**Review by mildas — COMMENTED** [review]
@yuumasato inconsistency in titles

File: rhel7/profiles/anssi_nt28_enhanced.profile (hunk 1 above)
Comment (yuumasato) [reply rhel7/profiles/anssi_nt28_enhanced.profile]: Thanks, addressed.

File: rhel7/profiles/anssi_nt28_high.profile (hunk 2 above)
Comment (yuumasato) [reply rhel7/profiles/anssi_nt28_high.profile]: Thanks, addressed.

**Comment by fduthilleul** [comment]
@yuumasato Should we not use ANSSI-BP-028 ? 
Please refer to: https://www.ssi.gouv.fr/uploads/2016/01/linux_configuration-fr-v1.2.pdf

**Comment by yuumasato** [comment]
> @yuumasato Should we not use ANSSI-BP-028 ?
> Please refer to: https://www.ssi.gouv.fr/uploads/2016/01/linux_configuration-fr-v1.2.pdf

@fduthilleul  The link in the description points to a page where both the French and English version are available.
https://www.ssi.gouv.fr/administration/guide/recommandations-de-securite-relatives-a-un-systeme-gnulinux/

I can point it to a specific version if preferred.

**Comment by yuumasato** [comment]
> @yuumasato Should we not use ANSSI-BP-028 ?
> Please refer to: https://www.ssi.gouv.fr/uploads/2016/01/linux_configuration-fr-v1.2.pdf

Sorry I misunderstood you.
In https://github.com/ComplianceAsCode/content/pull/6592/commits/5ea9fe70c78df6c4278aec71b9ab000a9884cea7, I have updated `ANSSI BP-028` to `ANSSI-BP-028`

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
lgtm, just small inconsistency in profile naming

File: rhel7/profiles/anssi_nt28_enhanced.profile [hunk 3]
```diff
@@ -1,9 +1,15 @@
 documentation_complete: true
 
-title: 'DRAFT - ANSSI DAT-BP28 (enhanced)'
+title: 'ANSSI-BP-028 (enhanced)'
 
-description: 'Draft profile for ANSSI compliance at the enhanced level. ANSSI stands for Agence nationale de la sécurité des
-    systèmes d''information. Based on https://www.ssi.gouv.fr/.'
+description: |-
+    This profile contains configurations that align to ANSSI-BP-28 at the enhanced hardening level.
```
[the remark this answers is not in the document]
Comment (yuumasato) [reply rhel7/profiles/anssi_nt28_enhanced.profile]: Very well noted. Updated in c111061

File: rhel7/profiles/anssi_nt28_enhanced.profile [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
 
-title: 'DRAFT - ANSSI DAT-BP28 (enhanced)'
+title: 'ANSSI-BP-028 (enhanced)'
 
-description: 'Draft profile for ANSSI compliance at the enhanced level. ANSSI stands for Agence nationale de la sécurité des
-    systèmes d''information. Based on https://www.ssi.gouv.fr/.'
+description: |-
+    This profile contains configurations that align to ANSSI-BP-028 at the enhanced hardening level.
+
+    ANSSI is the French National Information Security Agency, and stands for Agence nationale de la sécurité des systèmes d''information.
```
Comment (vojtapolasek) [inline rhel7/profiles/anssi_nt28_enhanced.profile:8]: sorry, just noticed. You have double apohstrophe in the full name of ANSSI

Comment (mildas) [reply rhel7/profiles/anssi_nt28_enhanced.profile:8]: Double apostrophe are also in description of other levels

Comment (yuumasato) [reply rhel7/profiles/anssi_nt28_enhanced.profile:8]: This is to escape the single quote.

Comment (mildas) [reply rhel7/profiles/anssi_nt28_enhanced.profile:8]: @yuumasato In generated guide, it shows as double. Copy-paste guide:
> ANSSI is the French National Information Security Agency, and stands for Agence nationale de la sécurité des systèmes d''information.

Comment (yuumasato) [reply rhel7/profiles/anssi_nt28_enhanced.profile:8]: Right, the behavior changed as the descriptions is not enclosed in single quotes anymore.

Comment (yuumasato) [reply rhel7/profiles/anssi_nt28_enhanced.profile]: Updated in c4b11df

**Comment by vojtapolasek** [comment]
looks good noqw, merning

**Review by vojtapolasek — APPROVED** [review]

**Review by mildas — APPROVED** [review]

**Comment by yuumasato** [comment]
Thank you for the review, gentleman.
A lot of issues were found, :)

**Pull request merged**